elementary/monitor/dbt_project/macros/get_adapter_type_and_unique_id.sql

```diff
@@ -12,4 +12,8 @@
 
 {% macro bigquery__get_adapter_unique_id() %}
     {{ return(target.project) }}
-{% endmacro %}
\ No newline at end of file
+{% endmacro %}
+
+{% macro trino__get_adapter_unique_id() %}
+    {{ exceptions.raise_compiler_error("Force failure and thus set warehouse_type=null since Trino is not currently supported within the report index.html") }}
+{% endmacro %}
```

Comment:
Kinda strange but without this, `warehouse_type` will get populated within `elementary_output.json` and since `elementary/monitor/data_monitoring/report/index.html` doesn't know how to handle `"warehouse_type": "trino"` the report will fail to render, forcing a failure here will ensure `warehouse_type` is set to `null` which will allow the HTML to render.

This issue has not been seen with other adapters like Athena, since Athena will fallback to the `default__get_adapter_unique_id` macro and since it has no `host` property, it will also fail, resulting in `"warehouse_type": null`

Reply:
I think we should probably fix this on the FE side on our end, I prefer this a bit to adding a hack, will look into it